include/InterpUtils.h

```diff
@@ -3,9 +3,25 @@
 
 #include <vector>
 
-#include "Util.h"
+#include "Vectors.h"
 
 namespace ale {
+
+  enum RotationInterpolation {
+    SLERP, // Spherical interpolation
+    NLERP // Normalized linear interpolation
```

Comment:
Can we explicitly define these?

Reply:
What do you mean?

Reply:
Similarly to how the PositionInterpolation is being defined as LINEAR = 0, SPLINE = 1. Can we do that with the RotationInterpolation modes?